### ISSUE TITLE
To Do list

### ISSUE THREAD

**Issue by AustinXII** [Zarel/honko-damagecalc#121]
I figured putting a todo list on the GitHub repo itself might be beneficial as it's much easier for developers to see what still needs to be done.
``1: Fix Mega's/alolan Pokémon search`` currently the calculator doesn't support searching for the Pokémons Mega or Alola forms. If you wanted to search for mega metagross for example, you'd have to type "metagross-mega". Changing the ``getSetOptions`` function in ``ap_calc.js`` should be the solution here 
``2: Fix best move auto selection`` currently when you change a pokemons EVs after already doing a calc it will redirect you off of the move that was previously the strongest. This is annoying people so adding a lock-move option might be a good thing to have, the best being if you double click on a move it stays highlighted.
``3: Add forme-change option`` a dropdown option that changes the Pokémons forme but not the set. This would require adding ``hasAltForm`` in ``pokedex_js``
``4: Imported sets only mode`` an option where only your imported sets show up on the left side of the calc, while the other showing sets from ``setdex_**.js``
``5: Fix stat dropping moves KO-Chance`` auto calculate ko chance with moves that drop a users stats. Pretty straight forward, add  ``dropsStats`` to any relevant move in ``moves.js`` then change any relevant code in ``ko_chance.js``

**Comment by AustinXII** [comment]
@MagiaSN feel free to tackle any of these if you're interested :)

**Comment by MagiaSN** [comment]
@AustinXII I will, just have something else to do recently :)

**Comment by AustinXII** [comment]
1 and 3 done in #157

**Comment by Ntonio36** [comment]
Can I join in ? 6 looks easy so I probably will be able do deal with it.

**Comment by AustinXII** [comment]
This is open to anyone :) feel free

**Comment by AustinXII** [comment]
2 was done as well

**Comment by Ntonio36** [comment]
Can you strike through the list elements that have been made, please ?

**Comment by AustinXII** [comment]
Yeah @Ntonio36 last time I tried the syntax didn’t work for somebody reason

**Comment by Ntonio36** [comment]
N°6 done in #180 and #181

**Comment by Ntonio36** [comment]
Number 8 done in #190

**Comment by Tyler-Hilbert** [comment]
Can we add in an export tool for all the presets?  Like it would be nice if I could just pull a pokemon off of damage calculator and put it into my team.

**Comment by Ntonio36** [comment]
Export option added in #217.
